esti/lakectl_test.go

```diff
@@ -10,10 +10,13 @@ import (
 
 	"github.com/stretchr/testify/require"
 	"github.com/treeverse/lakefs/pkg/api/apigen"
+	"github.com/treeverse/lakefs/pkg/graveler"
 )
 
 var emptyVars = make(map[string]string)
 
+const branchProtectTimeout = graveler.BranchUpdateMaxInterval + 1
```

Comment:
`graveler.BranchUpdateMaxInterval` is a `time.Duration `unit you should add `time.Second` instead of 1 to it